README.md

```diff
@@ -44,4 +44,48 @@ Basic product configuration, as referenced in the get started guides, can be fou
     implements bidirectional data synchronization between IDM and LDAP.
   * `/{version}/default/ig/basic-sample` - Contains configuration to deploy the simplest 
     possible IG server.
+    
+## Contents
+
+### 6.5 configurations   
+
+Directory                   | Contents
+----------------------------|-------------------------------------------
+----------------------------| **DEFAULT CONFIGURATIONS**
+\6.5\\**default**           | Default out of the box configurations to get started using our products.
```

Comment:
Any reason you're using backslashes in paths, rather than forward slashes?

Reply:
damn, thanks for spotting.  Didn't even notice I had done that :(